posthog/queries/test/__snapshots__/test_retention.ambr

```diff
@@ -1,3 +1,75 @@
+# name: TestFOSSRetention.test_day_interval_sampled
+  '
+  WITH actor_query AS
+    (WITH 'Day' as period,
+          NULL as breakdown_values_filter,
+          NULL as selected_interval,
+          returning_event_query as
+       (SELECT toStartOfDay(toTimeZone(toDateTime(e.timestamp, 'UTC'), 'UTC')) AS event_date,
+               pdi.person_id as target
+        FROM events e SAMPLE 1.0
+        INNER JOIN
+          (SELECT distinct_id,
+                  argMax(person_id, version) as person_id
+           FROM person_distinct_id2
+           WHERE team_id = 2
+           GROUP BY distinct_id
+           HAVING argMax(is_deleted, version) = 0) AS pdi ON e.distinct_id = pdi.distinct_id
+        WHERE team_id = 2
+          AND e.event = '$pageview'
+          AND toDateTime(e.timestamp) >= toDateTime('2020-06-10 00:00:00', 'UTC')
+          AND toDateTime(e.timestamp) <= toDateTime('2020-06-21 00:00:00', 'UTC')
+        GROUP BY target,
+                 event_date),
+          target_event_query as
+       (SELECT DISTINCT toStartOfDay(toDateTime(e.timestamp), 'UTC') AS event_date,
+                        pdi.person_id as target,
+                        [
+                          dateDiff(
+                              'Day',
+                              toStartOfDay(toDateTime('2020-06-10 00:00:00' , 'UTC')),
+                              toStartOfDay(toTimeZone(toDateTime(e.timestamp, 'UTC'), 'UTC'))
+                          )
+                      ] as breakdown_values
+        FROM events e SAMPLE 1.0
```

Comment:
oof this is going to be tricky, there are 2 samplings happening here, one for the target event query, one for the return event query, and these distinct IDs can be different, which is going to cause a lower retention rate the higher the sampling rate is, since tendency of mismatches would increase, even if we're just sampling by distinct IDs.

Reply:
Yeah so this is exactly why I wanted your eyes on here. Was thinking about this too. Unsure how to approach it. Fair to say that "retention shouldn't be sampled" btw

Reply:
We can try it out, but seems like it - I want to test the slider to see how inaccurate it gets. Either you can metabase it, or we can try on prod and revert slider if not okay? I'm fine either way, but yeah not confident this works well for retention